components/support/guid/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "sync-guid"
+version = "0.1.0"
+authors = ["Thom Chiovoloni <tchiovoloni@mozilla.com>"]
+license = "MPL-2.0"
+
```

```suggestion
edition = "2018"
```

`cargo fix` might help there.